src/modules/random.js

```diff
@@ -0,0 +1,22 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
+const generateRandomNumber = () => {
+  const randomNumber = MissionUtils.Random.pickNumberInRange(1, 9); // 1~9 중 랜덤 숫자
+  return randomNumber;
+};
+
+const generateStrikes = () => {
+  const numberArray = [];
+  while (numberArray.length < 3) {
+    let rn = generateRandomNumber();
```

Comment:
```suggestion
    let randomNumber = generateRandomNumber();
```
rn이라는 변수명은 의도를 들어내는데 충분하지 않다고 생각합니다.
프리코스 1주차 공통 피드백을 보면 `축약하지 않는다` 항목이 있는데, 그 부분에 위배됩니다!

Reply:
다음부터는 그 부분에 좀 더 유의하겠습니다! 감사합니다.